machines/request.ts

```diff
@@ -57,6 +57,7 @@ const model = createModel(
       GO_TO_RECEIVED_VC_TAB: () => ({}),
       CANCEL: () => ({}),
       DISMISS: () => ({}),
+      RESET: () => ({}),
```

Comment:
Since we are using machines/openIdBle/* files for BLE, do we need to make changes in machines/* files as well?

Reply:
Since Google nearby state machine is not entirely removed. When/If we switch back to google nearby, Without this event, It will fail.

And also to keep consistency between type of the both machines.